docs/ref/contrib/admin/index.txt

```diff
@@ -2827,6 +2827,37 @@ linked to the document in ``{% block dark-mode-vars %}``.
 
 .. _prefers-color-scheme: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme
 
+.. _extrabody:
+
+``extrabody`` template block
+=============================
+
+You can insert any custom HTML, JavaScript, or other content that needs
```

Comment:
```suggestion
=============================

.. versionadded:: 5.2

You can insert any custom HTML, JavaScript, or other content that needs
```

Reply:
Thanks for the feedback, I missed this part in the contribution docs. Added now.